### PR TITLE
Improving the test runner and updating SDK versions

### PR DESCRIPTION
**Pull request by foxish**
1. make the test runner scripts more robust
2. parameterize cluster name and org name across the board
3. verified that the test runner script succeeds

### CODE REVIEW

File: java-sdk-example/README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ This example application uses Signadot's Java SDK to create a sandbox and test t
 The test creates a sandbox in the setup phase, runs tests against it and deletes the sandbox upon completion of the tests. Heres' the command to run it:
 
 ```
+export SIGNADOT_ORG=...
```
Comment (jmsktm) [inline java-sdk-example/README.md:19]: Looks like it should be `SIGNADOT_ORG_NAME` based on the changes below.

File: python-sdk-example/test-requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
 pytest==6.2.5
-signadot-sdk==0.2.3
+signadot-sdk==0.2.5
```
Comment (jmsktm) [inline python-sdk-example/test-requirements.txt]: It's 0.2.4 for python sdk: https://pypi.org/project/signadot-sdk/

```suggestion
signadot-sdk==0.2.4
```

File: java-sdk-example/README.md [hunk 3]
```diff
@@ -16,6 +16,8 @@ This example application uses Signadot's Java SDK to create a sandbox and test t
...
 export SIGNADOT_API_KEY=...
+export SIGNADOT_CLUSTER_NAME=...
```
Comment (jmsktm) [inline java-sdk-example/README.md:21]: Arjun had once mentioned that he finds the prefix `Signadot` (eg. for signadot org) confusing. So maybe we can go without it while you're at it?

File: python-sdk-example/test-requirements.txt (hunk 2 above)
Comment (foxish) [reply python-sdk-example/test-requirements.txt]: Yeah, spotted this and then added strict error checking to the script also. Should be fixed in the latest commit.

File: java-sdk-example/README.md (hunk 3 above)
Comment (foxish) [reply java-sdk-example/README.md:21]: I am a bit worried that things like `CLUSTER_NAME` feel underspecified when we don't qualify with the `SIGNADOT_` prefix. WDYT?

File: java-sdk-example/README.md (hunk 1 above)
Comment (foxish) [reply java-sdk-example/README.md]: Done

File: java-sdk-example/README.md (hunk 3 above)
Comment (jmsktm) [reply java-sdk-example/README.md:21]: I feel `SIGNADOT_API_KEY` can remain as is as the users may be dealing with API keys for multiple things. But I'm less concerned about the other two. Thoughts, @arjun-sd ?

Comment (foxish) [reply java-sdk-example/README.md:21]: I don't have a strong opinion here, so, if you think we should remove it across the board, I can make that change.

**Review by jmsktm — APPROVED** [review]
LGTM. Feel free to make changes related to `SIGNADOT_` prefix as you see fit or as per Arjun's input (if any).

**Comment by foxish** [comment]
I've standardized to `SIGNADOT_ORG` everywhere because that's also consistent with the [CLI](https://docs.signadot.com/docs/cli#configuration).

**Pull request merged**